src/platforms/hubspot/schemas.ts

```diff
@@ -258,6 +247,7 @@ const notePropertiesSchema = z
   .object({
     hubspot_owner_id: hubspotIdSchema,
     hs_note_body: z.string(),
+    hs_timestamp: z.string(),
```

Comment:
just making sure, this is required? For some reason I thought hubspot would set it for you if not

Reply:
I have it `.partial()` below, not required in the updates/creates